src/TipBox.vue

```diff
@@ -168,14 +182,26 @@
           case 'definition':
             return '<i class="fas fa-atlas"></i>';
           default:
-            return '<i class="fas fa-exclamation"></i>';
+            return '';
         }
       },
       iconStyle() {
         if (this.iconSize) {
           return `fa-${this.iconSize}`;
         }
         return '';
+      },
+      noBackgroundStyle() {
+        if (this.noBackground) {
```

Comment:
I wonder if this will make this part less verbose:
```
return this.noBackground? 'no-background' : '';
```

Reply:
It's made it consistent to the other computed properties. For example,
```
      seamlessStyle() {
        if (this.isSeamless) {
          return 'seamless';
        }
        return '';
      },
      fontBlack() {
        if (this.light) {
          return 'font-black';
        }
        return '';
      },
```